test/plots/piePlotTests.ts

```diff
@@ -2,92 +2,140 @@
 
 describe("Plots", () => {
   describe("PiePlot", () => {
-    // HACKHACK #1798: beforeEach being used below
-    it("renders correctly with no data", () => {
-      let svg = TestMethods.generateSVG(400, 400);
-      let plot = new Plottable.Plots.Pie();
-      plot.sectorValue((d) => d.value);
-      assert.doesNotThrow(() => plot.renderTo(svg), Error);
-      assert.strictEqual(plot.width(), 400, "was allocated width");
-      assert.strictEqual(plot.height(), 400, "was allocated height");
-      svg.remove();
-    });
+    describe("Rendering", () => {
+      let svg: d3.Selection<void>;
+      let simpleDataset: Plottable.Dataset;
+      let simpleData: any[];
+      let piePlot: Plottable.Plots.Pie;
```

Comment:
in general I used the standard of calling the plot as `plot` (as opposed to `piePlot`), `scale` (as opposed to `linearScale`). No real good reason for this, except consistency. Comments?